src/sass/table/stories/table.stories.js

```diff
@@ -0,0 +1,263 @@
+export default { title: "Skin/Table" };
+
+export const base = () => `
+<div class="table">
+    <table>
+        <caption>
+            Video game consoles for sale.
+        </caption>
+        <thead>
+            <tr>
+                <th scope="col">Seller</th>
```

Comment:
Been a while since I checked this, but do we still need scope=col for such a simple table?

Reply:
> Been a while since I checked this, but do we still need scope=col for such a simple table?

@ianmcburnie , I checked with the `a11y` team about this. The general thought was, though it's not technically necessary, it's good to include as an explicit fallback to cover any gaps in various screen readers.

Reply:
Hmm, after doing some quick catch up research, I'm more of the opinion that they aren't necessary for simple/basic tables such as this. I think it's safe to remove them

https://adrianroselli.com/2017/11/a-responsive-accessible-table.html

Reply:
fixed